bitcoinspv/bootstrapping_test.go

```diff
@@ -0,0 +1,5 @@
+package bitcoinspv
+
+// TODO: unit tests on Bootstrap by using mocks
```

Comment:
remove if we are not testing anything here. We can create a separate issue for it